.docker/init.sh

```diff
@@ -50,6 +50,7 @@ elif [ "$ULOGGER_DB_DRIVER" = "sqlite" ]; then
   sqlite3 -init /var/www/html/scripts/ulogger.sqlite /data/sqlite/ulogger.db .exit
   sqlite3 -line /data/sqlite/ulogger.db "INSERT INTO users (login, password, admin) VALUES ('${ULOGGER_ADMIN_USER}', '\$2y\$10\$7OvZrKgonVZM9lkzrTbiou.CVhO3HjPk5y0W9L68fVwPs/osBRIMq', 1)"
   sed -i "s/^\$dbdsn = .*$/\$dbdsn = \"sqlite:\/data\/sqlite\/ulogger.db\";/" /var/www/html/config.php
+  chown -R nobody:nobody /data/sqlite
```

Comment:
The same command is already run on line 49. Is ownership changed in the meantime?

Reply:
Ah, I missed that earlier call. Yes, on my system, I needed the permission change.

Reply:
You are right. Could you delete the first invocation of `chown`?